pims/norpix_reader.py

```diff
@@ -150,12 +183,17 @@ def _verify_frame_no(self, i):
         if i >= self._image_count or i < 0:
             raise ValueError("Frame number is out of range: " + str(i))
 
+    def set_process_func(self, process_func):
```

Comment:
We are in the process of ripping that functionality back out to be replaced by `pipeline`, sorry for the poor communication on that.